official/utils/logs/logger.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, logging_dir):
     if not tf.gfile.IsDirectory(self._logging_dir):
       tf.gfile.MakeDirs(self._logging_dir)
 
+    self.run_date = datetime.datetime.now()
```

Comment:
Also, i think its better to keep as fewer internal field as possible. You can get the start timestamp by reading the log file.

Reply:
Will reading from the log file take longer?